test/utils/docker/centos6/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN yum clean all && \
     yum -y install epel-release && \
     yum -y install \
     acl \
-    asciidoc \
+    asciidoctor \
```

Comment:
This and the other containers in the same directory are no longer used to build docs, so they shouldn't be updated.

Reply:
Should we just remove asciidoc from there then? I mean it brings a LOT of stuffs with dblatex by default. This would lighten the images too.

Reply:
Leave them as-is for now. I'm going to be working on newer images and things can be cleaned up then.